package.json

```diff
@@ -45,6 +45,18 @@
       "<rootDir>/bootstrap/"
     ]
   },
+  "nodemonConfig": {
+    "execMap": {
+      "js": "babel-node -r dotenv/config"
+    },
+    "watch": [
+      "src",
+      "config",
+      "package.json",
+      "package-lock.json",
+      ".env"
+    ]
```

Comment:
Likewise, this replaces the `--ignore bootstrap/storage` "exclusion" with a list of "opt-in" directories & files to watch. Any time any of these change, the application will be automatically reloaded.